numpy/lib/function_base.py

```diff
@@ -3864,8 +3864,10 @@ def percentile(a,
                axis=None,
                out=None,
                overwrite_input=False,
-               interpolation="linear",
-               keepdims=False):
+               method="linear",
+               keepdims=False,
+               *,
+               interpolation=None):
```

Comment:
Interesting approach,  replace "interpolation" by "method" when used as positional arguments and only worry if "interpolation" is used as a keyword.